fiftyone/core/annotation/generate_label_schema.py

```
@@ -0,0 +1,436 @@
+        return _handle_str(
+            collection, field_name, is_list, settings, scan_samples
+        )
+
+    if isinstance(field, fof.BooleanField):
+        return _handle_bool(settings)
+
+    if isinstance(field, (fof.FloatField, fof.IntField)):
+        return _handle_float_or_int(settings)
+
+    if is_list:
+        raise ValueError(f"todo")
```

Pylint (4.0.3)

[warning] Using an f-string that does not have any interpolated variables

(W1309)